src/components/modules/dashboard/ui/pages/Dashboard.tsx

```
@@ -51,39 +52,53 @@ export const Dashboard = () => {
+          </TabsContent>
 
-        <DisputeAnalytics address={address} escrows={escrows || []} />
+          <TabsContent value="dispute-analytics" className="space-y-4">
+            <DisputeAnalytics address={address} escrows={escrows || []} />
+          </TabsContent>
+        </Tabs>
       </div>
     </>
   );
-};
+};
```

ESLint

[error] Insert `⏎`

(prettier/prettier)